aws/resource_aws_acm_certificate.go

```diff
@@ -0,0 +1,237 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/acm"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+	"log"
+	"time"
+)
+
+func resourceAwsAcmCertificate() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsAcmCertificateCreate,
+		Read:   resourceAwsAcmCertificateRead,
+		Update: resourceAwsAcmCertificateUpdate,
+		Delete: resourceAwsAcmCertificateDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
I tried it out, looks like this leads to some interesting interactions: When using the fully automatic validation flow (e.g. with route53 and the `acm_certificate_validation` resource), the `acm_certificate` resource completes first (with a `PENDING_VALIDATION` status), then the route53 record gets added and `acm_certificate_validation` waits until ACM issues the certificate. Since the `acm_certificate` resource isn't refreshed at that point, it stays in status `PENDING_VALIDATION`. After the next terraform run, this state switches to `ISSUED`. 

I haven't seen an immediate impact but I'd guess this can lead to some interesting surprises down the road :)